integrationExamples/gpt/permutiveRtdProvider_example.html

```diff
@@ -180,11 +199,6 @@
                       gender: 'm',
                       keywords: 'a,b',
                       data: [
-                        {
-                          name: 'www.dataprovider1.com',
-                          ext: { taxonomyname: 'iab_audience_taxonomy' },
-                          segment: [{ id: '687' }, { id: '123' }]
-                        },
```

Comment:
two things left: circleci needs to run; add a segtax here. Can be the permutive contextual segtax or the iab contextual (don't use the audience bc that is user.data)